Lecture3/Task1/utils/OutputGenerator.java

```diff
@@ -0,0 +1,13 @@
+package utils;
+
+import product.Product;
+
+import java.io.PrintWriter;
+
+public class OutputGenerator {
+	public static void createProducts(Product product, PrintWriter productOutput) {
```

Comment:
method `create Products ` don't describe what this method really do

moreover you don't need this class& just use String.format() in FileWriter

Reply:
done